charts/gardener/controlplane/charts/runtime/templates/controller-manager/configmap-componentconfig.yaml

```diff
@@ -31,6 +31,11 @@ data:
       kubeconfig: /etc/gardener-controller-manager/kubeconfig/kubeconfig
       {{- end }}
     controllers:
+      {{- if .Values.global.controller.config.controllers.bastion }}
```

Comment:
Maybe you can also extend https://github.com/gardener/gardener/blob/master/docs/concepts/controller-manager.md#control-loops with information to this controller, and https://github.com/gardener/gardener/blob/master/example/20-componentconfig-gardener-controller-manager.yaml with example configuration